src/scoreboard/scoreboard.js

```diff
@@ -0,0 +1,122 @@
+// Javascript
+import 'bootstrap/dist/js/bootstrap.min';
+
+// Stylesheets
+import 'bootstrap/dist/css/bootstrap.min.css';
+import './scoreboard.css';
+
+
+$(document).ready(function ()
+{
+    let scoreboardUrl = 'scoreboard.html';
+    let scoreboardApiUrl = 'http://scoreboard.test';
+    let roomName = getUrlParam('room');
+    let timeBetweenUpdates = 30; // seconds
+
+    $('[id^="page-"]').hide();
+
+    if (roomName == undefined) {
+        $('#page-home').show();
+    } else {
+        $('#page-room').show();
+
+        getRoomInfo(roomName);
+    }
+    
+    $('#search-room').click(e => {
+        e.preventDefault();
+
+        window.location = scoreboardUrl + '?room=' + $('#search-room-input').val();
+    });
+
+    $('#create-room').click(e => {
+        e.preventDefault();
+        
+        $.ajax({
+            method: 'POST',
+            url: scoreboardApiUrl + '/rooms',
+        }).done(data => {
+            window.location = scoreboardUrl + '?room=' + data.name;
+        });
+    });
+
+    function getRoomInfo(name) {
+        $.ajax({
+            method: 'GET',
+            url: scoreboardApiUrl + '/rooms/' + name,
+        }).done(data => {
+            $('#room-name').text(data.name);
+
+            getTeams(name);
+            setTimer();
+
+            setInterval(function () {
+                getTeams(name);
+                setTimer();
+            }, timeBetweenUpdates * 1000);
+        }).fail(error => {
+            window.location = scoreboardUrl;
+        });
+    }
+
+    function setTimer() {
+        let timeLeft = timeBetweenUpdates;
+
+        $('#scoreboard-timer').html(timeBetweenUpdates);
+
+        let x = setInterval(() => {
+            $('#scoreboard-timer').html(--timeLeft);
+
+            if (timeLeft <= 0) {
+                clearInterval(x);
+            }
+        }, 1 * 1000);
```

Comment:
Should this be a parameter?

Reply:
I don't believe so, since it counts down per second. The only thing you want to change is the refresh rate.